components/list/ContribList.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+  <p align="center">
+    <a
+      v-for="c in contribs"
+      :key="c.url"
+      class="contr-a"
+      :href="c.fmn_url || c.html_url"
+      target="_blank"
+    >
+      <img :style="`width: ${size || '10%'}`" class="contr-image" :src="c.avatar_url" />
+    </a>
+  </p>
+</template>
+
+<script>
+export default {
+  name: 'ContribList',
+  props: ['contribs', 'size']
```

Comment:
propları `required` ve `default` model belirlenen yaklaşımda yazmanı tavsiye ederim. Ek olarak buradaki size prop'unun ismini değiştir. Bu componente yukardan geçtiğin `size` prop'unun neyin size'ı olduğu hiç anlaşılmıyor. `avatarSize` gibi bir isimlendirme tercih edebilirsin.